src/scenario_tests.rs

```diff
@@ -699,22 +957,30 @@ mod dst_tests {
         let initial_state = arrange_initial_state(
             &initial_state_young_elders(),
             &[
-                ParsecVote::ExpectCandidate(CANDIDATE_1).to_event(),
+                ParsecVote::ExpectCandidate(CANDIDATE_1_OLD).to_event(),
+                CANDIDATE_INFO_VALID_PARSEC_VOTE_1.to_event(),
+                ParsecVote::CheckResourceProof.to_event(),
                 ParsecVote::Online(CANDIDATE_1).to_event(),
                 ParsecVote::CheckElder.to_event(),
             ],
         );
 
+        let description = "Get Parsec ExpectCandidate then Online (Elder Change) RemoveElderNode for wrong elder,\
```

Comment:
I guess it'd be good to stick to the 100 char limit here, but if rustfmt isn't fussed, who am I to disagree!? :)

Reply:
rustfmt cannot format this string, so should not trust it.
I thought we were 120

Reply:
[Think it's 100 at the moment](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#max_width)